### ISSUE TITLE
Dev toolbar létrehozása

### ISSUE THREAD

**Issue by tmichel** [kir-dev/korok#28]
Egy csak _development_ módban elérhető toolbarra/szekcióra lenne szükség, ahol fejlesztési időben on the fly lehetne állítani az alkalmazás bizonyos paramétereit. Ez nagyban megkönnyítené a tesztelést, lehetőséget biztosítana, hogy több felhasználóval és több jogkörrel lehessen egyes funkciókat kipróbálni.

Alapvető követelmények:
- minden oldalról könnyen elérhető
- lehet állítani az éppen bejelentkezett user-t
- a user kiválasztására valami intuitívabb felületen történik (pl autocomplete list)
- lehessen specifikus tulajdonságra szűrni és úgy user-t választani
- lehessen egyéb jogokat is állítani
  - milyen körnek a vezetője
  - milyen _role_-okkal rendelkezik

Minderre már van egy kezdeményezés a `DummyAuthorization` osztályban, de egyelőre csak újrafordítással és a adatbázisban keresgéléssel lehet felhasználót váltani.

**Comment by vbalazs** [comment]
erre vállalkozott végül valaki?

related: https://trello.com/c/Ry1L9I9E/25-dev-toolbar-a-korokhoz

**Comment by tmichel** [comment]
nem, de jó lenne, ha nem mi csinálnánk meg. ez egy szép összetett feladat, megismerné az illető a webréteget legalább. 

@vbalazs listán hirdessük még!

**Comment by vbalazs** [comment]
július 5-én küldtél ki erről egy levelet már listára, egyedül Ksisu érdeklődött, de ő max csak augusztus végétől tudna vele foglalkozni (szvsz akkor sem )

szerk.: áttettem másik milestoneba, mert ez valószínűleg blokkolná a 2.6-os release-t és azt még a hónapban szeretnénk élesíteni.

**Comment by tmichel** [comment]
Némi előrelépés. A jelenlegi wicketes alkalmazásban nagyjából megvalósult. Egyelőre csak virid alapján. 

Releváns commitok: https://github.com/kir-dev/korok/compare/bcc2263...8a3d6e2

**Használat:**

`DEVELOPMENT` módban látogass el a `/dev` oldalra és adj meg egy user id-t.
